libminifi/include/utils/FlatMap.h

```diff
@@ -147,6 +147,22 @@ class FlatMap{
     return data_.rbegin()->second;
   }
 
+  const V& at(const K& key) const {
```

Comment:
These new `FlatMap` member functions are not used in this PR any longer, so I would remove them and their tests.  But they are probably going to be useful later, so if you want to keep them, that's fine, too.

Reply:
I think we should keep them since they are already written and they make the class more in line with the C++20 std::map/std::unordered_map interface.